lib/TwoFactorAuth.php

```diff
@@ -81,7 +82,8 @@ public function getCode($secret, $time = null)
      */
     public function verifyCode($secret, $code, $discrepancy = 1, $time = null, &$timeslice = 0)
     {
-        $timetamp = $this->getTime($time);
+    	$code = (string)$code; //even though the code is numeric, it must be compared as string, so, to be sure...
```

Comment:
Stumbled upon this when I type-hinted in my internal method with `int`... I discovered the hard way that `hash_equals('123', 123)` is false.